seed/seed.json

```diff
@@ -406,7 +406,7 @@
                 },
                 {
                     "name": "MaximumAdNotificationsPerDay=100/MaximumInlineContentAdsPerHour=8/MaximumInlineContentAdsPerDay=40/AdServingVersion=2",
-                    "probability_weight": 50,
+                    "probability_weight": 20,
```

Comment:
Feature weights will be set to the feature default as:

1. DoesMatchIntentChildSegments: 1.0
2. DoesMatchIntentParentSegments: 1.0
3. DoesMatchInterestChildSegments: 1.0
4. DoesMatchInterestParentSegments: 1.0
5. AdLastSeenHoursAgo: 1.0
6. AdvertiserLastSeenHoursAgo: 1.0
7. Priority: 1.0